napalm/nxos/nxos.py

```diff
@@ -44,7 +44,9 @@
 
 class NXOSDriverBase(NetworkDriver):
     """Common code shared between nx-api and nxos_ssh."""
-    def commit_config(self):
+    def commit_config(self, message=''):
+        if not message:
+            raise NotImplementedError('Commit message not implemented for this platform')
```

Comment:
I think we should do the same for `nxos_ssh`? @ktbyers

Reply:
There is no `commit_config()` in `nxos_ssh`.